handlers/oam_handler.go

```diff
@@ -190,6 +220,16 @@ func (h *Handler) OAMComponentDetailsHandler(rw http.ResponseWriter, r *http.Req
 	}
 }
 
+// swagger:route GET /api/oam/{type}/{name}/{id} PatternsAPI idOAMComponentDetailByID
+// Handles GET requests for component details for OAM objects
+//
+// Returns details of a workload/trait/scope by given name and id in the URL
+//
+// {type} being of either trait, scope, workload; registration of adapter capabilities.
+//
```

Comment:
Thank you. Examples will be quite helpful, if you have any. // @utkarsh-pro @Revolyssup

Reply:
I'm not very expert of this area, @Revolyssup can you share some e.g. that could be added to description?

Reply:
@utkarsh-pro or @Revolyssup, a couple examples here would help a good deal.

Reply:
@leecalcote what kind of example needs to be added? To be precise, an example of the response needs to be added or an example of the endpoint needs to be added?

Reply:
@utkarsh-pro thanks for clarifying. Examples of endpoints are ideal; a set of examples that provide something of a template such that a contributor could plugin the name of an object (by taking a guess) and hit the endpoint even though not all endpoints are exhaustively listed in the docs.